posts/how-to-commit-code-to-a-git-repository-with-github-desktop.md

```diff
@@ -0,0 +1,144 @@
+---
+title: How to commit code to a Git repository with GitHub Desktop
+author: marek-zaluski
+tags: [tutorial, git, github, open source]
+description: Learn how to create commits in a Git repository with GitHub Desktop
+image: https://storage.googleapis.com/sourcegraph-assets/learn/headers/sourcegraph-learn-04.png
+publicationDate: September 13, 2021
+imageAlt: Sourcegraph Learn
+type: posts
+---
+
+Committing code with Git allows you to version control your own work, collaborate with other developers, and contribute to open source projects. GitHub Desktop is an application that provides a graphical interface to interact with Git repositories on your computer, providing an alternative to using Git on the command line. It can also clone, push, and pull remote repositories hosted on GitHub and elsewhere.
+
+In this tutorial, we’ll go through how to commit code to a Git repository using GitHub Desktop.
+
+## Prerequisites
+
+Be sure that you have GitHub Desktop installed. You can install it by going to the [GitHub Desktop website](https://desktop.github.com/) and selecting the version of the software relevant to your operating system.
+
+To follow this tutorial, you should already have a local Git repository on your computer and have made some changes that you want to commit. If you don’t yet have a Git repository on your computer, you can review the official documentation for "[Getting a Git Repository](https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository)".
```

Comment:
Except for code, quotes should be smart / curly quotes as in `“` and `”`. The full stop should be inside of the end quote.